UI.AutomationTests/Video/RepTests.cs

```diff
@@ -0,0 +1,72 @@
+using BookingsApi.Contract.V2.Enums;
+using UI.PageModels.Pages.Video.Participant;
+using VideoApi.Contract.Responses;
+
+namespace UI.AutomationTests.Video
+{
+    public class RepTests : VideoWebUiTest
+    {
+        private ConferenceDetailsResponse _conference;
+
+        [Description("Book a hearing. Log into hearing with a Rep. Use Rep Skip self test journey")]
+        [Test]
+        [Category("video")]
+        [Category("coreVideo")]
```

Comment:
I don't think this needs to be in the coreVideo since I don't believe it's essential for a quick run to check everything works but rather a nice-to-have category